doc/source/operating.rst

```diff
@@ -263,4 +263,52 @@ your file system). The procedure to restore is no more complicated than initial
 deployment: restore your index and storage backups, reinstall Gnocchi if
 necessary, and restart it.
 
+How to clear Gnocchi data
+=========================
+
+If you ever want to start fresh or needs to clean Gnocchi data, this can be
+easily done. You need to clean the storage data, the incoming data and the
```

Comment:
Just a suggestion: `You need to clean the measure, aggregate, and index storage`